PowerEditor/src/Parameters.h

```diff
@@ -923,6 +924,8 @@ struct NppGUI final
 	DarkModeConf _darkmode;
 
 	LargeFileRestriction _largeFileRestriction;
+
+	TCHAR _lastUsedDir[MAX_PATH];

```

Comment:
Will do, but let me explain why I put it there - plugin considerations. By putting `_lastUsedDir` in the middle of the struct, I'm forcing every plugin that interacts with that struct to make a bugfix patch, because the newly inserted member shifts everything after it. Putting new members at the end does not change the layout of preexisting members.

Reply:
> By putting `_lastUsedDir` in the middle of the struct, I'm forcing every plugin that interacts with that struct to make a bugfix patch [...]

Adding a new member in *any* position is already a breaking change. If a plugin allocates insufficient memory for the bigger `struct`, the result will be an out-of-bounds write, leading to memory corruption if not a crash